src/cljs/rems/application.cljs

```diff
@@ -141,7 +141,7 @@
                            {:application-id application-id}
                            {:catalogue-items catalogue-items}))}))
 
-(defn- submit-application [app description application-id catalogue-items items licenses]
+(defn- submit-application [app description application-id catalogue-items items licenses prev-validation]
```

Comment:
Why the name `prev-validation`?

Reply:
it's the validation from the previous save. or would you prefer just `validation`?

Reply:
Just wondering why to submit an application you need to know the previous validation.

Reply:
Save returns the validation errors, but dynamic submit just returns `{:errors [:form-not-valid]}`

I don't think we should add validation messages to the return schema of dynamic /command

Reply:
I'm not sure I get what you mean.